laws/src/main/scala/cats/laws/discipline/TraverseFilterTests.scala

```diff
@@ -0,0 +1,42 @@
+package cats
+package laws
+package discipline
+
+import org.scalacheck.{Prop, Arbitrary}
+import Prop._
+
+trait TraverseFilterTests[F[_]] extends TraverseTests[F] with FunctorFilterTests[F] {
+  def laws: TraverseFilterLaws[F]
+
+  def traverseFilter[A: Arbitrary, B: Arbitrary, C: Arbitrary, M: Arbitrary, X[_]: Applicative, Y[_]: Applicative](implicit
+    ArbFA: Arbitrary[F[A]],
+    ArbXB: Arbitrary[X[B]],
+    ArbYB: Arbitrary[Y[B]],
+    ArbYC: Arbitrary[Y[C]],
+    ArbAXOB: Arbitrary[A => X[Option[B]]],
+    ArbBYOC: Arbitrary[B => Y[Option[C]]],
+    M: Monoid[M],
+    EqFA: Eq[F[A]],
+    EqFC: Eq[F[C]],
+    EqM: Eq[M],
+    EqXYFC: Eq[X[Y[F[C]]]],
+    EqXFA: Eq[X[F[A]]],
+    EqXFB: Eq[X[F[B]]],
+    EqYFB: Eq[Y[F[B]]]
+  ): RuleSet = {
+    new RuleSet {
+      def name: String = "collect"
```

Comment:
Good catch, @mikejcurry! This was left over from when I had used `Collect` as this type class name at first.